modules/pages/client/views/volunteering.client.view.html

```diff
@@ -84,13 +84,10 @@ <h2>Volunteering</h2>
           <p>If you're still not sure and just want to chat to some humans, introduce yourself on our <a href="https://meta.trustroots.org/">volunteer forum</a>. All are welcome and we'll be happy to help you get started.</p>
         </li>
 
-        <li>
-          <strong><em>ONE MORE THING!</em></strong>
-          <p><a href="https://github.com/Trustroots/trustroots/wiki/Volunteering">All the same stuff on our Github wiki...</a></p>
-        </li>
-
       </ul>
 
+      <small><a class="btn btn-small btn-primary pull-right" href="https://github.com/Trustroots/trustroots/edit/master/modules/pages/client/views/volunteering.client.view.html" target="_blank">Edit this page<i class="icon-github icon-lg"></i></a></small>
```

Comment:
btn-xs or btn-sm class should do the trick without `small` tag. :thumbsup:

Let's add also `rel="noopener noreferrer"` for security because of _blank.

https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

Reply:
Great, I'm adding some more buttons to pages that can use, plus some more improvements to texts (e.g. removed one link to GH wiki).